src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -957,13 +964,12 @@ def visit_Index(self, node: ast.Index):
 
     def visit_Subscript(self, node: ast.Subscript):
         assert isinstance(node.ctx, (ast.Load, ast.Store))
-        index = self.visit(node.slice)
+        index = tuple(sl.value for sl in gt_utils.listify(self.visit(node.slice)))
```

Comment:
This fails when slice is a unary operator, e.g.,
```python
q2 = (qin[-1, 0, 0]
```
with error,
```
AttributeError: 'UnaryOpExpr' object has no attribute 'value'
```

Reply:
Good catch! I switched to using `ast.literal_eval` for subscript indices, since they should always be literals (for now).